docs/iot/guides/iot-api.md

```diff
@@ -13,6 +13,11 @@ The following table lists the IoT-specific Tizen platform API group.
 
 | IoT-specific API                                             | Description                                                  |
 | ------------------------------------------------------------ | ------------------------------------------------------------ |
-| [Things SDK API](things-api.md) | The SmartThings&trade; Things SDK APIs help you to create new Things applications on IoT devices. For the full specification, see the [Things SDK API Reference](https://developer.tizen.org/dev-guide/things-sdk). |
+| [Things SDK API](things-api-5.0.md) | The SmartThings&trade; Things SDK APIs help you to create new Things applications on IoT devices. For the full specification, see the [Things SDK API Reference](https://developer.tizen.org/dev-guide/things-sdk/5.0). |
```

Comment:
Please advice me here, based on my limited knowledge, I think the abbreviation must be written as:

The SmartThings™(ST) Things SDK APIs helps to create new Things applications on IoT devices.

Reply:
Where is the abbreviation? Do you mean we need to add "(ST)"?

Reply:
Ok, understand, thanks.